### ISSUE TITLE
Autofill pop up inconsistent compared to previous notifications api

### ISSUE THREAD

**Issue by 05nelsonm** [bitwarden/mobile#802]
## Describe the Bug
The new api for displaying autofill as an overlay in the app username/password field (especially with browsers when at a login page of a website), doesn't always show up. The old notification api was FAR more consistent.

Is there a way to give users the option to choose either or?

**Comment by ContentPersonality2** [comment]
Fully agreed! The previous system was far superior for a user like myself. I had the old system set to persistent notification to conserve battery life and to keep the app from being shutdown in the background. The new system regularly doesn't work for me and is very inconsistent and messy. I would have much preferred this be an opt-in or at the very least have the old system be an opt-in. I don't understand why the old system was removed, surely it wouldn't have been much effort if any to continue supporting it alongside this new inconsistent system.

In short please bring back the old system as an option, especially the consistent notification version because that was the most reliable.

**Comment by kspearrin** [comment]
Duplicate https://github.com/bitwarden/mobile/issues/784

We are working on a new alternative way to autofill until we can figure out whey this popup overlay isnt working right for some people.
